src/apps/ENGINE/src/internal_functions.cpp

```diff
@@ -432,7 +432,7 @@ DATA *COMPILER::BC_CallIntFunction(uint32_t func_code, DATA *&pVResult, uint32_t
 
     long slen, slen2;
     char sVarName[64];
-    char sBuff2[2];
+    char8_t utf8_character[5];
```

Comment:
maybe better to use std::string here? that would simplify the code a tiny bit, I think